<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="home">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>845</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="windowmpl" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-10</y>
        <width>611</width>
        <height>491</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>611</width>
         <height>451</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="mplvl">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
       </layout>
      </widget>
     </widget>
     <widget class="QTextEdit" name="thisworks">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>611</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Controller">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Axis&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>68</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Host&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>68</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Port&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="axisMenu">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Axis</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="axisVal">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="hostVal">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>100</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="portVal">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>100</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="hostMenu">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>250</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Host</string>
      </property>
     </widget>
     <widget class="QPushButton" name="portMenu">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>250</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Port</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="connectionStat">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>370</y>
        <width>431</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="updateConnect">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>470</y>
        <width>211</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Connection</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connectPS4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>310</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Manual Control</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Notes"/>
    <widget class="QWidget" name="NotSure"/>
   </widget>
   <widget class="QListWidget" name="list">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>91</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="end">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>520</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Finish</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuChoose_Plot">
    <property name="title">
     <string/>
    </property>
    <addaction name="actionAltitude"/>
    <addaction name="actionDistance"/>
   </widget>
   <addaction name="menuChoose_Plot"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAltitude">
   <property name="text">
    <string>Altitude</string>
   </property>
  </action>
  <action name="actionDistance">
   <property name="text">
    <string>Distance</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
